.github/workflows/enos-release-testing-oss.yml

```diff
@@ -0,0 +1,43 @@
+name: enos_release_testing_oss
+
+on:
+  repository_dispatch:
+    types:
+      - enos_release_testing_oss
```

Comment:
type should match the event_type in `ci.hcl` https://github.com/hashicorp/vault/pull/18056/files#diff-d51ad13228611273f3804158eadd6b38eb7fe8eb46d5e84b94e6416a15c63709R178 so should be `enos-release-testing-oss`